丰富网站内容

wold9168 opened the pull request:
- **Add LOGO to the Home Page**
- **Move the Example Files into the Specific Folder**
- **更新了 homepage 页面**
- **更新了 README 文件**
- **修改了主页、导航栏，重命名 LOGO 文件**
- **添加了一些页面**
- **修改了仓库和网站的许可证**
- **添加了多侧边栏支持**

wold9168 committed:
Add LOGO to the Home Page

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
Move the Example Files into the Specific Folder

Move the example files created by VitePress into the specific folder
`docs/examples/`.

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
更新了 homepage 页面

将 homepage 页面重命名为 about，并更新了其中的信息。

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
更新了 README 文件

更新了 README 文件。许可证是否更改仍需商榷。

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
修改了主页、导航栏，重命名 LOGO 文件

修改了主页的内容。修改了导航栏的标题，同时去除了导航栏的无效项。

重命名了 LOGO 文件。

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
添加了一些页面

添加了站点共建
(contribute.md)、新闻列表(news.md)、技术交流企划(techex.md)相关的页面，并添加了相应的侧边栏。修订了
about.md 页面。

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
修改了仓库和网站的许可证

修改了仓库和站点的许可证，移除了 LICENSE 文件，为 README
和站点页脚添加了有关许可证的描述。

新取用的许可证为 CC BY 4.0 International。

Signed-off-by: wold9168 <moewold@outlook.com>

wold9168 committed:
添加了多侧边栏支持

添加了对新闻列表的多侧边栏支持

Signed-off-by: wold9168 <moewold@outlook.com>

Cherrling commented on LICENSE:
cc 的协议有这种格式化的英文版吗，考虑加进来？
直接把 LICENSE 扬了是不是有点粗暴

wold9168 committed:
修改 about.md 的措辞

根据 github.com/Cherrling/hitlug/pull/3#discussion_r2264695733 的内容，对
about.md 的措辞进行修改。

Signed-off-by: wold9168 <moewold@outlook.com>
